docs/_getting-started/index.md

```diff
@@ -4,89 +4,202 @@ permalink: docs/getting-started
+
+## Connect to Geth with IPC or RPC
+
+You can interact with Geth in two ways: Directly with the node using the JavaScript
+console over IPC, or connecting to the node remotely over HTTP using RPC.
+
+IPC allows you to do more, especially when it comes to creating and interacting
+with accounts, but you need direct access to the node.
+
+RPC allows remote applications to access your node but has limitations and security
+considerations, and by default only allows access to methods in the `eth` and `shh`
+namespaces. Find out how to override this setting [in the RPC docs](rpc/server#http-server). 
+
+## Using IPC
+
+### Connect to console
+
+You can connect to the IPC console in two ways. First, when you start the node:
 
 ```shell
-geth --syncmode "light"
+geth --goerli --syncmode "light" --rpc --signer=<CLEF_LOCATION>/clef.ipc console
 ```
 
-### Javascript Console
+The node also outputs sync information, so using the console in the same terminal can be difficult.
 
-Once you have an account and Geth is running, you can interact with it by opening another
-terminal and using the following command to open a JavaScript console:
+You can also open a console on the node from another terminal using:
 
 ```shell
-geth attach
+geth attach <IPC_LOCATION>
+```
+
+### Check account balance
+
+```javascript
+web3.fromWei(eth.getBalance("<ADDRESS_1>"),"ether")
+```
+
+Getting the balance of an account does not require a signed transaction,
+so Clef does not ask for approval, and Geth returns the value.
+
+### Send ETH to account
+
+Send 0.01 ETH from the account that you added ETH to with the Görli faucet,
+to the second account you created:
+
+```javascript
+eth.sendTransaction({from:"<ADDRESS_0>",to:"<ADDRESS_1>", value: web3.toWei(0.01,"ether")})
 ```
 
-In the console you can issue any of the Geth commands, for example, to list all the
-accounts on the node, use:
+This action does require signing, so Clef prompts you to approve it, and if you
+do, asks you for the password you are sending the ETH from. 
+If the password is correct, Geth proceeds with the transaction.
 
-```js
-> eth.accounts
+To check, get the account balance of the second account:
+
+```javascript
+web3.fromWei(eth.getBalance("<ADDRESS_1>"),"ether")
 ```
 
-You can also enter the console directly when you start the node with the `console` command:
+## Using RPC
+
+### Connect to RPC
+
+You can use standard HTTP requests to connect to a Geth node using the RPC APIs, for example:
+
+```shell
+curl -X POST http://<GETH_IP_ADDRESS>:8545 --data \
+    '{"jsonrpc":"2.0",
+    "method":"<API_METHOD>",
+    "params":[],
+    "id":1}' \
+    -H "Content-Type:application/json"
+```
+
+### Check account balance
+
+```shell
+curl -X POST http://<GETH_IP_ADDRESS>:8545 --data \
+    '{"jsonrpc":"2.0",
+    "method":"eth_getBalance",
+    "params":["<ADDRESS_1>","latest"],
+    "id":1}' \
+    -H "Content-Type:application/json"
+```
```

Comment:
These examples would look better if you put the RPC payload on a single line, e.g.

```
curl -X POST http://<GETH_IP_ADDRESS>:8545 -H "Content-Type:application/json" --data \
    '{"jsonrpc": "2.0", "id": 1, "method": "eth_getBalance", "params": ["<ADDRESS_1>", "latest"]}'
```